cmake/dependencies.cmake

```diff
@@ -95,9 +95,10 @@ fetch_dep(GTest
 set(ADA_TESTING OFF)
 set(ADA_TOOLS OFF)
 set(ADA_BENCHMARKS OFF)
+set(ADA_INCLUDE_URL_PATTERN OFF)
 fetch_dep(ada
   REPO https://github.com/ada-url/ada
-  TAG v2.7.3)
+  TAG v3.2.2)
```

Comment:
This should be 3.2.4 I suppose?

Reply:
Cc @rockwotj

Reply:
This file is unused and vestigial. Will remove in a followup PR